### PR TITLE
Fix #1519

### LINKED ISSUE #1519: Broken wiki links in documentation
There are ~many~ 4 links to https://github.com/monicahq/monica/wiki/... though there is no more wiki.

### PR DESCRIPTION
**Pull request by ChucklesTheBeard**
Fix links in documentation

Closes #1519

### CODE REVIEW

File: docs/installation/docker.md [hunk 1]
```diff
@@ -110,7 +110,3 @@ Note that uploaded files, like avatars, will disappear when you
 restart the container. Map a volume to
 `/var/www/monica/storage/app/public` if you want that data to persist
 between runs. See `docker-compose.yml` for examples.
-
-#### Other documents to read
-
-[Connecting to MySQL inside of a Docker container](https://github.com/monicahq/monica/wiki/Installation-with-Docker:-connection-to-MySQL)
```
Comment (asbiin) [inline docs/installation/docker.md]: Well @djaiss, any idea where this doc is ?

**Comment by asbiin** [comment]
@ChucklesTheBeard Thank you !
Could you please also fix the "Contribute as a developer" link on docs/readme.md ? In fact the "contribute.md" directory should be renamed with "contribute" ...

**Comment by ChucklesTheBeard** [comment]
Fixed as requested.

File: docs/installation/docker.md (hunk 1 above)
Comment (ChucklesTheBeard) [reply docs/installation/docker.md]: I can grab the copy from https://github.com/monicahq/monica/blob/40bc3373d8536ae5d05fe2c676567ff9e89ae12c/docs/database/connecting.md but IDK if it was updated while it was on the wiki.

**Pull request merged**